contracts/Vault.vy

```diff
@@ -1582,32 +1584,35 @@ def _assessFees(strategy: address, gain: uint256) -> uint256:
         / SECS_PER_YEAR
     )
 
-    # Only applies in certain conditions
-    strategist_fee: uint256 = 0
-    performance_fee: uint256 = 0
+    self.managementDebt +=  management_fee
+
+    if gain == 0:
+        return 0
+
+    # if total performance fee is greater than 100% then this will cause an underflow
+    assert self.strategies[strategy].performanceFee + self.performanceFee <= MAX_BPS
```

Comment:
It won't as we going to merge this
https://github.com/yearn/yearn-vaults/pull/348

Reply:
in order to safe. so make more judgments

Reply:
You have to keep it gas efficient the other checks are covering it

Reply:
I see.
now delete it. test may fail. may update test